src/guidelines/theme/style.ts

```diff
@@ -323,3 +323,4 @@ const breakpoints: Breakpoints = {
 };
 
 export { theme, breakpoints };
+export type { BorderRadius };
```

Comment:
you might need to make **BorderRadius** available in **"src/guidelines/theme.index.ts"**  -> **"export { theme, breakpoints } from "src/guidelines/theme/style";**

Reply:
Done